### DISCUSSION TITLE
Migrating capabilities

### DISCUSSION THREAD

**Discussion by davidstanke** [dora-team/dora.dev#131] [General]
I'm starting this thread to discuss the job of migrating the capabilities content from cloud.google.com/devops to dora.dev. @michelleirvine and @sapientcoffee have gotten us off to a great start. :)

@sapientcoffee thank you for porting the Cultural capabilities. Do you plan to tackle another category next? Is there anything standing in your way of getting that done? LMK how I can help!

**Comment by sapientcoffee** [comment]
I will migrate the capabilities, I should get though some while traveling on Friday. The next wave will be process

**Comment by davidstanke** [comment]
Update: the "process" articles have been migrated (thanks Rob!).

And FYI for anyone who sees this discussion: step 1 is simply to migrate the content, without regard for design. It's a bit messy! Then we'll clean them up... although I'm hopeful we can do that somewhat in parallel, because I hope that the bulk of the work will happen in the templates and CSS, rather than in individual articles.

**Reply by davidstanke** [reply]
All the capabilities are in -- check out draft.dora.dev/capabilities 😍

I plan to do a pass on styles/navigation tomorrow, and then I think we can swap the top-level nav to point to these instead.

**Comment by davidstanke** [comment]
The capabilities are live! Now we need to work on i18n. But I'm going to close this discussion, since we're treating that as a distinct (high priority!) project.
